lib/address_geocoder/MapApiManager

```diff
@@ -0,0 +1,20 @@
+# Map API Manager don't Know HOW to do every action
+# know WHAT to do and WHO to do
+# know address_geocoder requirement
+
+require 'map_apis/*'
```

Comment:
Should really only require the map_api that the user has chosen (can require in method)
e.g. `require "map_apis/#{api}"